spec/commands_spec.rb

```diff
@@ -208,45 +211,37 @@ def rakefile_source
     end
 
     it "should create a new gem prefixed with nesta-plugin" do
+      @command.stub(:specify_gem_dependency)
       @command.should_receive(:system).with('bundle', 'gem', @gem_name)
-      begin
-        @command.execute
-      rescue Errno::ENOENT
-        # This test is only concerned with running bundle gem; ENOENT
-        # errors are raised because we didn't create a real gem.
-      end
+      @command.execute
     end
 
     describe "after gem created" do
       def create_gem_file(*components)
         path = File.join(@plugins_path, @gem_name, *components)
-        FileUtils.makedirs(File.dirname(path))
         File.open(path, 'w') { |f| yield f if block_given? }
         path
       end
 
+      def lib_path(*parts)
+        File.join(@gem_name, 'lib', *parts)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.